vesc_driver/src/throttle_interpolator.py

```diff
@@ -123,8 +122,12 @@ def _process_servo_command(self, msg):
 
 # Boilerplate node spin up.
 if __name__ == "__main__":
+    import sys
+
+    car_name = "/car" if len(sys.argv) < 2 else sys.argv[1]
```

Comment:
Usually we'd use the ROS parameter server to get parameters like this. Then you can use `rospy.get_param("~car_name", "/car")` to get the parameter, then in the launch file you can use:

```
<node pkg="vesc_driver" type="throttle_interpolator.py" name="throttle_interpolator">
<param name="car_name" value="$(arg car_name)" />
</node>
```

I find having named parameters instead of positional args a little nicer personally.